### PR TITLE
update roary to latest version in conda. 

### PR DESCRIPTION
**Pull request by Takadonet**
update roary to latest version in conda. 
Allow for collection identifier or filename to be used instead of everything tempDIGITAL

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

**Comment by bgruening** [comment]
@Takadonet can you please look also add https://github.com/galaxyproject/tools-iuc/pull/2518 and merge if necessary. 

Thanks!

File: tools/roary/roary.xml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-      <requirement type="package" version="3.10.2">roary</requirement>
+      <requirement type="package" version="3.12.0">roary</requirement>
     </requirements>
     
     <command detect_errors="exit_code"><![CDATA[
         
-        #for $counter, $gff in enumerate($gff_input.gffs)
-          cp $gff temp${counter}.gff &&
+        #for $gff in $gff_input.gffs
+          cp $gff '$gff.element_identifier'.gff &&
```
Comment (bgruening) [inline tools/roary/roary.xml]: not sure this works, can you use `'${gff.element_identifier}.gff'`

Comment (Takadonet) [reply tools/roary/roary.xml]: It does work but added {} for it to be consistent.

**Comment by Takadonet** [comment]
Reviewed changes in #2518 for Roary tool. To address comments on that merge request, we need to use a cp command instead of ln -s for input sets since Roary will follow the soft link and use **dataset_#.dat** instead of the element_identifier.

**Comment by bgruening** [comment]
Thanks!

**Pull request merged**